### ISSUE TITLE
Inert elements not hidden from screen readers

### ISSUE THREAD

**Issue by gtritchie** [GoogleChrome/inert-polyfill#15]
When a tree is marked inert, it is still visible to screen readers.

Should be given `aria-hidden="true"` attribute otherwise a screen-reader user can navigate to and manipulate things they shouldn't be able to (e.g. when a modal dialog is covering the rest of the UI they can get at things "under" the modal dialog).

Can see this on the simple demo page, also in our project: https://github.com/rstudio/rstudio/issues/6075

**Comment by samthor** [comment]
So I think in general we are unlikely to implement this in this polyfill. I will call it out in the README under Limitations.

The WICG polyfill is more exhaustive re: accessibility (but changes the `tabindex` etc of all inert elements, which takes longer / performs more mutations).

**Comment by gtritchie** [comment]
For our case, I simply add `aria-hidden="true"` to the same element I was adding `inert` to and that does the trick. Ditto when I remove `inert`, also remove `aria-hidden`. Not necessary to apply it to everything in the tree.

Thanks for the reply.

**Comment by samthor** [comment]
Thanks, I'll add that suggestion to the README soon. I think we don't want to go down the automated route as that brings us much closer to the WICG polyfill (which is slower + arguably more _correct_).
